src/db.test.ts

```diff
@@ -533,3 +533,20 @@ Deno.test(
     db.close();
   },
 );
+
+Deno.test("autoCommit", {
```

Comment:
We should use a more descriptive name like “autoCommit property reflects database state”. Also, can we move the test to be collocated next to the tests for `changes` and `totalChanges` 😅